docs/domains/Domain.md

```diff
@@ -80,6 +82,7 @@ An element representing a cluster in the domain configuration.
 | `allowReplicasBelowMinDynClusterSize` | Boolean | If true (the default), then the number of replicas is allowed to drop below the minimum dynamic cluster size configured in the WebLogic domain home configuration. Otherwise, the operator will ensure that the number of replicas is not less than the minimum dynamic cluster setting. This setting applies to dynamic clusters only. |
 | `clusterName` | string | The name of this cluster. Required |
 | `clusterService` | [Kubernetes Resource](#kubernetes-resource) | Customization affecting ClusterIP Kubernetes services for the WebLogic cluster. |
+| `maxConcurrentStartup` | number | The maximum number of managed servers that the operator will start in parallel for the cluster in response to a change in replicas count for the cluster. If more managed servers need to be started, the operator will wait until a managed server to be in Ready and Running state before starting the next managed server. A value of 0 (the default) means all managed servers will start in parallel. |
```

Comment:
managed servers -> Managed Servers (globally)
will wait until a managed server to be in Ready and Running state -> will wait until a Managed Server is in the `Ready` and `Running` state.
(Hmmm, in the WLS docs, https://docs.oracle.com/en/middleware/standalone/weblogic-server/14.1.1.0/start/server_life.html#GUID-B1352C2D-351C-47C0-BE40-CA109367263C, there is a `RUNNING` state but not a `READY` state. Maybe these docs are not applicable.

Reply:
Thanks. The "Ready and Running state" should be for the managed server pod. So this is now updated to read 
"If more Managed Servers need to be started, the operator will wait until a Managed Server pod is in the `Ready` and `Running` state before starting the next Managed Server. "